### ISSUE TITLE
save() got an unexpected keyword argument 'force_insert' with django-rest-framework?

### ISSUE THREAD

**Issue by falkowich** [django-ldapdb/django-ldapdb#99]
It looks like there is a problem for me with django-rest-framework and django-ldapdb save() function.

In the save object force_insert=False is included, and things goes haywire :)
This was a fugly fix for me, any ideas what goest wrong and what I can do to not hardcode base.py?

```
models/base.py
65c65
<     def save(self, using=None):
---
>     def save(self, using=None, *args, **kvargs):

```
This is the installed packages..

pip freeze
```
Django==1.9.9
django-ldapdb==0.6.0
djangorestframework==3.4.6
pkg-resources==0.0.0
pyldap==2.4.25.1
```
And the Traceback
```
Traceback (most recent call last):
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/rest_framework/views.py", line 474, in dispatch
    response = self.handle_exception(exc)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/rest_framework/views.py", line 434, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/rest_framework/views.py", line 471, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/falk/dev/ldapdbtest/d1xldap/views.py", line 34, in post
    serializer.save()
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/rest_framework/serializers.py", line 192, in save
    self.instance = self.create(validated_data)
  File "/home/falk/dev/ldapdbtest/d1xldap/serializers.py", line 12, in create
    return Device.objects.create(**validated_data)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/db/models/manager.py", line 122, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/falk/env/ldapdbtest/lib/python3.5/site-packages/django/db/models/query.py", line 401, in create
    obj.save(force_insert=True, using=self.db)
TypeError: save() got an unexpected keyword argument 'force_insert'
```

--
Regards Falk

**Comment by ikreb7** [comment]
It seems that it is the same problem like #54.

**Comment by falkowich** [comment]
I'll try to mock sumfink up later today?

But I'm not a coder per se, I'm just an infosec/network fella with a python interest :)
So perhaps I'm doing sumfink wrong..

--
Regards Falk

**Comment by falkowich** [comment]
I made a small example with drf, ldapdb here -> https://github.com/falkowich/django_ldapdb_test
And an vagrant with an openldap .

Is that sumfink that you can use as an example @ikreb7?

--
Regards Falk

**Comment by rbarrois** [comment]
Indeed, the ``ldapdb.models.Model.save()`` signature is incorrect.

Thanks for the report!

**Comment by rbarrois** [comment]
Fixed as 25ae984.
